src/main/java/org/veupathdb/service/sequence/Main.java

```diff
@@ -22,4 +36,15 @@ protected ContainerResources newResourceConfig(Options options) {
 
     return out;
   }
+
+  @Override
+  protected void postCliParse(Options opts) {
```

Comment:
i think we should discuss this CLI thing.  i dont know if any of the other services do this.  interesting.

Reply:
@steve-fischer-200 We use this in OSI as well.